teams/account-management/README.md

```diff
@@ -1,5 +1,54 @@
 # Account Management
 
+## Setup your local dev environment
+
+To get started on local development and testing:
+
+1. **Get your `buffer-dev` environment setup**
+  → https://github.com/bufferapp/buffer-dev
+
+2. **Install the latest version of `yarn`**
```

Comment:
@hharnisc - do we use yarn? Is there any other step we are missing like npm install?

Reply:
We do have a dependency on yarn yes :smile: 

We've got a `./dev setup` command that is useful here and installs project deps

Reply:
@hharnisc - Can't we move yarn to "the container" as we did with npm `docker exec -it bufferdev_core-authentication-service_1 npm install`. Could we run something similar with yarn?